### PR TITLE
Fix Linter configuration

### PR DESCRIPTION
**Pull request by mu-hun**
tslint is deprecate tool. Update dev Dependencies for use ESLint

### CODE REVIEW

**Comment by x86chi** [comment]
Screen Shot 2020-08-20 at 8 43 37 PM

I think it's okay to use type aliases if you just intend to alias them.

**Comment by riemannulus** [comment]
> I think it's okay to use type aliases if you just intend to alias them.
Is this change get passed test? btw I'm reference this code from [here](https://medium.com/@michaelolof/typescript-mix-yet-another-mixin-library-29c7a349b47d). is it got same meaning with your proposal?

**Comment by mu-hun** [comment]
The test passes. In the case of type aliases refer to the original type. 

Type Alias in TypeScript does not appear as a name when checking static type checking, But refers to the type originally designated as an alias.

1*ziWdISgNakU3t2HyKkPpPg

> One difference is that interfaces create a new name that is used everywhere. **Type aliases don’t create a new name**
> 
> [Interfaces vs. Type Aliases  | Advanced Types · TypeScript](https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases)

I think it is correct to use Type Alias if you are not going to extend it after defining the type.

You are already know [Interface can be extend with redeclare](https://www.typescriptlang.org/docs/handbook/advanced-types.html#interfaces-vs-type-aliases) by open-close principle?

```ts
interface Window {
  title: string
}

interface Window {
  ts: import("typescript")
}

const src = 'const a = "Hello World"';
window.ts.transpileModule(src, {});
```

**Comment by riemannulus** [comment]
> I think it is correct to use Type Alias if you are not going to extend it after defining the type.

I understand. it seems be ok.

but we removed this code from https://github.com/riemannulus/HeartOfCrown/pull/12 when library chaged. and I think this conversation is out of point on this PR.

btw, can you resolve conflicts again? sorry about that 😢

**Comment by mu-hun** [comment]
I've fixed conflict.

**Comment by riemannulus** [comment]
I fix broken CI on your PR. Can you check it is what you want?

**Review by mu-hun — COMMENTED** [review]
Approve ✅

File: __tests__/core/basic_card.spec.ts [hunk 1]
```diff
@@ -11,7 +15,7 @@ describe('Basic Card', () => {
   describe('Apprentice Maid', () => {
     test('name', () => {
       const apprenticeMaid = new ApprenticeMaid();
-      expect(apprenticeMaid.name).toBe("Apprentice Maid");
+      expect(apprenticeMaid.name).toBe('Apprentice Maid');
```
Comment (mu-hun) [inline __tests__/core/basic_card.spec.ts:18]: is it meaningful change to singe qoute?

Comment (riemannulus) [reply __tests__/core/basic_card.spec.ts:18]: idk about that... I just run prettier autofix. 😅

**Comment by riemannulus** [comment]
Thanks for contribution!

**Pull request merged**